### ISSUE TITLE
interactive console with dap-chrome/dap-firefox (javascript)

### ISSUE THREAD

**Issue by mstoss** [emacs-lsp/dap-mode#497]
Hello, is it possible to have in dap-mode the interactive debug console in emacs like the same interactive console we have in firefox or chrome ?
The only way i found is to have this is to use a breakpoint at the end of my javascript source and to execute "dap-debug".
If I start the REPL without using the breakpoint the REPL in not connected with the brower.

Thank you for your help.

**Comment by yyoncho** [comment]
have you tried M-x dap-ui-repl?

**Comment by mstoss** [comment]
Yes, i tried it. But i need to put a break point at the end of my js file otherwise dap-ui-repl tell me "There is no stopped debug session".
